index.html

```diff
@@ -8,11 +8,10 @@
         <div class="header">
             <a class="logo logo_center" href="/"><img class="logo__image logo__image_big"></a>
             <div class="background__border"></div>
-        </div>
-        
-        <div class="cover">
-            <h1 class="title title_left title_big">passion a responsitive email template</h1>
-            <a class="link link_button_orange link_text_big">download</a>
+            <div class="cover">
+                <h1 class="title title_left title_big">passion a responsitive email template</h1>
+                <a class="link link_button_orange link_text_big">download</a>
+            </div>
         </div>
```

Comment:
So, if you planning to set image as a background for block `header` how you will use `cover` block?

Reply:
I was thinking about it... So, in this case it`s no need to use 'cover' block

Reply:
Please remove it